spec/models/measurement_spec.rb

```diff
@@ -16,14 +16,9 @@
 #
 # You can contact the authors by email at <info@habitatmap.org>
 
-require 'spec_helper'
+require 'rails_helper'
 
 describe Measurement do
-  it { is_expected.to validate_presence_of :value }
-  it { is_expected.to validate_presence_of :longitude }
-  it { is_expected.to validate_presence_of :latitude }
-  it { is_expected.to validate_presence_of :time }
```

Comment:
Have you consider rewriting them to something like 
```
 it "is invalid without a value" do
   m = Measurement.new(value: nil)
   m.valid?
   expect(m.errors[:value]).to include("can't be blank")
 end
```
Or maybe we don't need to test validations in newer rails versions?

Reply:
I think these tests are meaningless. The fact that `validates :....` works that way is tested by rails. I believe declarative code of a framework should not be tested.

It's like testing that 1 + 1 == 1. I trust ruby to give me a working `+`